src/widgets-service.js

```diff
@@ -36,7 +36,8 @@ WidgetsService.prototype.newCategoryModel = function (attrs, layer) {
   var dataviewModel = this._dataviews.createCategoryModel(layer, {
     type: 'category',
     column: attrs.column,
```

Comment:
BTW, as @javisantana pointed, we should change that value name to ```aggregation_column```, just to be consistent if later is it exposed in the viz.json.